src/IGDB.php

```diff
@@ -286,25 +286,34 @@ public function getGame($gameId, $fields = ['*'])
     /**
      * Search games by name
      *
-     * @param string $search
+     * @param null|string $search
      * @param array $fields
      * @param integer $limit
      * @param integer $offset
      * @param string $order
+     * @param array $filters
+     * @param string $expand
      * @return \StdClass
      * @throws \Exception
      */
-    public function searchGames($search, $fields = ['*'], $limit = 10, $offset = 0, $order = null)
+    public function searchGames($search = null, $fields = ['*'], $limit = 10, $offset = 0, $order = null, array $filters = [], $expand = '')
     {
         $apiUrl = $this->getEndpoint('games');
 
-        $params = array(
-            'fields' => implode(',', $fields),
-            'limit' => $limit,
+        $csvFields = implode(',', $fields);
+
+        $params = [
+            'fields' => $csvFields,
+            'limit'  => $limit,
+            'filter' => $filters,
+            'expand' => $expand,
             'offset' => $offset,
-            'order' => $order,
-            'search' => $search,
-        );
+            'order'  => $order
+        ];
+
+        if ($search !== null) {
```

Comment:
The `/games` endpoint will return totally different results if you query it with `&search=` (empty search.